component/componenthelper/doc.go

```diff
@@ -0,0 +1,17 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package componenthelper assists with the creation of a new component.Component.
+package componenthelper
+ 
```

Comment:
```suggestion

```

Text files must end in a newline character.  A stray space made its way in here!  If possible, configure your editor to ensure text files always end in a newline character.

Reply:
Great catch, Anthony!